openapi/components/schemas/RuleActionScheduleReminder.yaml

```
@@ -0,0 +1,30 @@
+allOf:
+  - $ref: ./RuleAction.yaml
+  - type: object
+    properties:
+      role:
+        type: string
+        enum:
+          - all
+          - renewal
+          - trial-end
+        description: Role of Reminder (available only on Order events, other events should use `all`).
+      schedule:
+        type: object
+        description: Schedule instructions collection.
```

nitpick: [noun string](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#noun-strings-or-noun-stacks)
```suggestion
        description: Collection of schedule instructions.
```